app/api/orders.py

```diff
@@ -125,32 +125,47 @@ def before_get_object(self, view_kwargs):
             attendee = safe_query(self, TicketHolder, 'id', view_kwargs['attendee_id'], 'attendee_id')
             view_kwargs['order_identifier'] = attendee.order.identifier
 
+        order = safe_query(self, Order, 'identifier', view_kwargs['order_identifier'], 'order_identifier')
+
+        if not has_access('is_coorganizer_or_user_itself', event_id=order.event_id, user_id=order.user_id):
+            return ForbiddenException({'source': ''}, 'Access Forbidden')
+
+
     def before_update_object(self, order, data, view_kwargs):
```

Comment:
According to the handbook, only status should be allowed to change by the organizer, once the order has been created, please implement that